configure

```diff
@@ -467,6 +482,9 @@ STRIPOPTS=$STRIPOPTS
 # Feature settings
 SUPPORT_CGROUPS=$SUPPORT_CGROUPS
 USE_INITGROUPS=$USE_INITGROUPS
+SUPPORT_CAPABILITIES=$SUPPORT_CAPABILITIES
```

Comment:
If `SUPPORT_CAPABILITIES` is 1 then `-lcap` needs to be added to `LDFLAGS` and `TEST_LDFLAGS` _if_ they haven't been specified manually. See for example the tests under the heading "`# Check whether sanitizers can be used for tests`" (that's only dealing with `TEST_LDFLAGS` but the general principle should be the same for `LDFLAGS`).